IpmiFeaturePkg/Library/IpmiCommandLib/IpmiCommandLibNetFnSensor.c

```diff
@@ -0,0 +1,83 @@
+/** @file
+  IPMI Command - NetFnSensor.
+
+  Copyright (c) 2018 - 2021, Intel Corporation. All rights reserved.<BR>

```

Comment:
Is this Intel code from somewhere? if not use the MSFT copyright

Reply:
Good catch, I missed that when creating the file.